src/Features/OnboardingAutomateTaxes.php

```diff
@@ -67,4 +68,11 @@ public static function on_onboarding_profile_completed() {
 	private static function automated_tax_is_supported() {
 		return in_array( WC()->countries->get_base_country(), \OnboardingTasks::get_automated_tax_supported_countries(), true );
```

Comment:
I think we'll need to remove the `\` before `OnboardingTasks` to use the correct namespace.  I see the following error:

<img width="654" alt="Screen Shot 2020-10-07 at 12 05 39 PM" src="https://user-images.githubusercontent.com/10561050/95357268-552a7e00-08d0-11eb-9e0a-cd3216dd50ea.png">

Reply:
Whoops. Thanks!